api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -167,8 +207,7 @@ export const getSurveyTemplateS3KeySQL = (surveyId: number, templateId: number):
   }
 
   const sqlStatement: SQLStatement = SQL`
-    SELECT
-      key
+    SELECT *
```

Comment:
This function's name is outdated now that the sql is returning the whole row and not just the key.
Same with the JSdoc description and the log labels.

Reply:
Maybe call this something like `getOccurrenceSubmissionSQL`

Reply:
Also, `surveyId` can be removed from the params/null check as its not used anywhere in the query

Reply:
I would also change `tempalteId` to `occurrenceSubmissionId`